src/quic/node_quic.cc

```diff
@@ -92,9 +93,10 @@ void QuicInitSecureContext(const FunctionCallbackInfo<Value>& args) {
 
   InitializeSecureContext(sc, side);
 
-  // TODO(@jasnell): Throw a proper node.js error with code
-  if (!crypto::SetGroups(sc, *groups))
-    return env->ThrowError("Failed to set groups");
+  if (!crypto::SetGroups(sc, *groups)) {
+    Isolate* isolate = env->isolate();
+    isolate->ThrowException(ERR_QUIC_CANNOT_SET_GROUPS(isolate));
```

Comment:
```suggestion
    THROW_ERR_QUIC_CANNOT_SET_GROUPS(env);
```

(afaik that should work)